test/powerflow.jl

```diff
@@ -37,4 +37,25 @@ import ExaPF: PowerSystem
             @test convergence.n_iterations == 2
         end
     end
+    @testset "Deport only powerflow on device $device" for device in DEVICES
+        polar = PolarForm(pf, CPU())
+        jac = ExaPF._state_jacobian(polar)
+        precond = ExaPF.LinearSolvers.BlockJacobiPreconditioner(jac, npartitions, CPU())
+        # Retrieve initial state of network
+        x0 = ExaPF.initial(polar, State())
+        uk = ExaPF.initial(polar, Control())
+        p = ExaPF.initial(polar, Parameters())
+
+        @testset "Powerflow solver $(LinSolver)" for LinSolver in [ExaPF.LinearSolvers.DirectSolver]
+            algo = LinSolver(precond)
+            xk = copy(x0)
+            nlp = ExaPF.ReducedSpaceEvaluator(polar, xk, uk, p;
+                                              ε_tol=tolerance, linear_solver=algo)
+            nlp_pf = ExaPF.ReducedSpaceEvaluator(nlp, device)
+            convergence = ExaPF.update!(nlp, uk; verbose_level=ExaPF.VERBOSE_LEVEL_NONE, nlp_pf = nlp_pf)
```

Comment:
I am not 100% sure with this API, but I have no better idea to propose at the moment. I think we could discuss on how to improve the API after merging this PR.

Reply:
> Before merging this PR, I would suggest doing a rebase to get rid of the `fix` and `Merge` commits. What do you think ?
Of course we do rebase 😊